src/shims/dir.props

```diff
@@ -25,6 +25,6 @@
   </ItemGroup>
 
   <PropertyGroup>
-  	<GenFacadesOutputPath>$(IntermediateOutputPath)facades/</GenFacadesOutputPath>
+  	<GenFacadesOutputPath>$(BaseIntermediateOutputPath)/shims/facades/</GenFacadesOutputPath>
```

Comment:
I would expect you still need the Configuration encoded into this path otherwise multiple configurations will conflict with each other.